dockerfiles/entrypoints/webpack.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/sh
+
```

Comment:
We use a `common.sh` script to be ran in all the containers that it's usually called here. I'm not sure if it OK to add it here since this image won't have Python installed and may break when using `pip install whatever` in the common script.

If we are not calling it here, we should remove it from the `volumen:` in the `webpack` service.

Reply:
Yeah, this container is closer to Azurite -- it doesn't contain any application code, it's an external service. I'd agree we probably don't want to run common.sh here.